fdbserver/Resolver.actor.cpp

```diff
@@ -441,7 +440,7 @@ ACTOR Future<Void> resolveBatch(Reference<Resolver> self,
 				reply.tpcvMap.clear();
 			} else {
 				std::set<uint16_t> writtenTLogs;
-				if (shardChanged || reply.privateMutationCount) {
+				if (shardChanged || toCommit->getAllMessages().size() > 1) {
```

Comment:
Can you confirm that "toCommit->getAllMessages()" returns private/metadata mutations only?

Reply:
It should.

Reply:
Ok.